### ISSUE TITLE
Theme Showcase: Theme switch modal window

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#52564]
This modal is a wall of text, and I find it intimidating/a little scary. I have to read the options carefully to make sure I understand what's happening here, and I'm still not sure I do. The cognitive load factor is high.

<img width="509" alt="Screen Shot 2021-05-03 at 5 41 12 PM" src="https://user-images.githubusercontent.com/2124984/117020519-816e7780-acc4-11eb-9457-a88c657f5bd9.png">

1. Go to the Theme Showcase
2. Click on a new theme from the "Recommended themes" list and click Activate
3. You'll see the above modal

I know this was introduced as a stop-gap because theme switching is hard. :) Is there a way to make it feel less scary? Or clearer? @kathrynwp maybe you have some thoughts on this from the user's perspective; I worry I'm making a mountain out of a molehill.

**Comment by kwight** [comment]
I'm inclined to call it low priority – except when the confusion is maximum for the Blank Canvas theme ("I thought it was blank?!...").

<img width="590" alt="Screen Shot 2021-05-04 at 12 31 43 PM" src="https://user-images.githubusercontent.com/349751/117063892-4f681000-acda-11eb-9031-c9ca8854f71c.png">

**Comment by kathrynwp** [comment]
Could re perhaps simplify the wording to avoid repeating the theme name?  Like:

"Keep my homepage content"

"Move my homepage content to a page draft"

**Comment by kwight** [comment]
See also https://github.com/Automattic/wp-calypso/issues/52792

**Comment by kwight** [comment]
Closing in favour of https://github.com/Automattic/wp-calypso/issues/52792
